backend/services/esweService.ts

```diff
@@ -43,6 +44,14 @@ export const DEFAULT_SKILL_LEVELS = {
   ],
 }
 
+export const DEFAULT_LEARNER_HISTORY = {
+  total: '0',
```

Comment:
Yeah, the tag component requires you to pass the content as a string. It's meant for short messages rather than counts I guess https://design-system.service.gov.uk/components/tag/

Reply:
The service shouldn't really know about the view. A service is normally data and domain centric.
This issue seems more like a view layer concern. Perhaps this should be formatted in either the template or the route handler